### PR TITLE
Create neealdon3.json

### PR DESCRIPTION
**Pull request by neealdon3**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Review by neealdon3 — COMMENTED** [review]
good

File: domains/neealdon3.json
Comment (Lukas-Batema) [inline domains/neealdon3.json:1]: Where is the stuff for the domain?

**Review by Lukas-Batema — CHANGES_REQUESTED** [review]

**Review by WilliamDavidHarrison — CHANGES_REQUESTED** [review]
Please refer to @Lukas-Batema's review.

**Review by WilliamDavidHarrison — DISMISSED** [review]
I cannot reach your website.

**Comment by neealdon3** [comment]
> I cannot reach your website.

dang nab it the url is eternal-network.github.io/public

**Review by DEV-DIBSTER — APPROVED** [review]

**Comment by neealdon3** [comment]
it doesn't work

**Pull request merged**